config/ecs/config.go

```diff
@@ -70,6 +70,9 @@ func Configure(p *config.Provider) {
 				Type:      "github.com/upbound/provider-aws/apis/iam/v1beta1.Role",
 				Extractor: common.PathARNExtractor,
 			},
+			"load_balancer.target_group_arn": {
```

Comment:
```suggestion
			"load_balancer.target_group_arn": config.Reference{
```

Could you please update it as above to be consistent?